src/cmd/restart.go

```diff
@@ -16,21 +17,33 @@ func RestartSpotify(flags ...string) {
 
 	switch runtime.GOOS {
 	case "windows":
-		exec.Command("taskkill", "/F", "/IM", "spotify.exe").Run()
-		if isAppX {
-			ps, _ := exec.LookPath("powershell.exe")
-			exe := filepath.Join(os.Getenv("LOCALAPPDATA"), "Microsoft", "WindowsApps", "Spotify.exe")
-			flags = append([]string{"-NoProfile", "-NonInteractive", `& "` + exe + `" --app-directory="` + appDestPath + `"`}, flags...)
-			exec.Command(ps, flags...).Start()
-		} else {
-			exec.Command(filepath.Join(spotifyPath, "spotify.exe"), flags...).Start()
+		isRunning := exec.Command("tasklist", "/FI", "ImageName eq spotify.exe")
```

Comment:
![image](https://user-images.githubusercontent.com/9348108/165545754-f165952f-7af3-48a3-bd30-7f78389430e9.png)
?

Reply:
Supposedly it works with the quotation marks (`tasklist /FI "ImageName eq Spotify.exe"`)
![image](https://user-images.githubusercontent.com/77577746/165548331-159b1db9-bbda-462d-af55-db3fa86afee4.png)
but will break if I add them in the file, works if it is without the quotes
![image](https://user-images.githubusercontent.com/77577746/165548395-fae66560-cc83-42fd-a9f2-ea1bda810a64.png)